internal/resources/audit_log_export_schema.go

```diff
@@ -7,16 +7,16 @@ import (
 func AuditLogExportSchema() schema.Schema {
 	return schema.Schema{
 		Attributes: map[string]schema.Attribute{
-			"id":                     stringAttribute([]string{required}),
+			"id":                     stringAttribute([]string{computed}),
 			"organization_id":        stringAttribute([]string{required}),
```

Comment:
We also figure out which field is requireReplace and which is not

Reply:
@aniket-Kumar-c export does not have update/delete endpoints, so that is why I didn't set any field with `requireReplace`

Reply:
yes, if the resource doesn't update that means, when we are changing  any field it should not show, it will make update call in terraform plan, so ideally all configurable marked as requiresReplace as changing the field suggest users that it will delete the resource and create new resource, check allowlist as the reference

Reply:
Addressed in 6fea4c906b1a62b8076ff2346b8ad5aa6c5c7374

Reply:
As discussed on slack, `requiresReplace` not needed.  I reverted the commit that added it.